pkg/3scale/amp/operator/backend.go

```diff
@@ -104,9 +104,7 @@ func (o *OperatorBackendOptionsProvider) setBackendRedisOptions(b *component.Bac
 }
 
 func (o *OperatorBackendOptionsProvider) setReplicas(b *component.BackendOptionsBuilder) {
-	if o.APIManagerSpec.HighAvailability != nil && o.APIManagerSpec.HighAvailability.Enabled {
-		b.ListenerReplicas(2)
-		b.WorkerReplicas(2)
-		b.CronReplicas(2)
-	}
+	b.ListenerReplicas(int32(*o.APIManagerSpec.Backend.ListenerSpec.Replicas))
+	b.WorkerReplicas(int32(*o.APIManagerSpec.Backend.WorkerSpec.Replicas))
+	b.CronReplicas(int32(*o.APIManagerSpec.Backend.CronSpec.Replicas))
```

Comment:
Cron jobs replicated? what is the purpose?

Reply:
Having availability of pods in case one fails

Reply:
But a cronjob, by definition, does not need pod replica. Just replication controller that ensures 1 pod will be alive and then cron can perform the task every X secs

Reply:
If you ensure you have two you don't need to wait for the pod to be relocated/recreated or miss the "execution interval"

The same happens with requests (although I agree with requests it makes much more sense than this specific case). It's better to have already available pods than missing some requests.

I don't think it does any harm to allow the configurability of it.

Reply:
It is a cronjob to reschedule failed tasks. You can wait for the pod to be relocated/recreated and you can miss the execution interval.

Allowing multiple replicas may impact if resource limits are defined. 

Anyway, up to you